### PR TITLE
chore: export types from logger module

### PR DESCRIPTION
**Pull request by bartoszherba**
[TFAL-71](https://alokai.atlassian.net/browse/TFAL-71)

[TFAL-71]: https://alokai.atlassian.net/browse/TFAL-71?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/sdk/src/modules/loggerModule/index.ts [hunk 1]
```diff
@@ -1,32 +1,3 @@
...
-  });
-
-  return {
-    connector: {
-      ...loggerWithMetadata,
-    },
-  } satisfies Module;
-};
+export { loggerModule } from "./module";
+export * from "./utils";
```
Comment (dawid-ziobro) [inline packages/sdk/src/modules/loggerModule/index.ts:2]: There's no need to export `utils`, this is internal only.

**Review by dawid-ziobro — APPROVED** [review]

File: .changeset/eighty-turkeys-marry.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@vue-storefront/sdk": major
```
Comment (Fifciu) [inline .changeset/eighty-turkeys-marry.md:2]: Why major?

File: packages/sdk/src/modules/loggerModule/index.ts (hunk 1 above)
Comment (bartoszherba) [reply packages/sdk/src/modules/loggerModule/index.ts:2]: Fixed

File: .changeset/eighty-turkeys-marry.md (hunk 2 above)
Comment (bartoszherba) [reply .changeset/eighty-turkeys-marry.md:2]: Changed to patch, I thought it does not matter if we are on the prerelease mode.

**Pull request merged**